.circleci/config.yml

```diff
@@ -18,7 +18,8 @@ jobs:
         name: Build
         command: |
           python3.8 -m venv venv/
-          venv/bin/pip install --upgrade -r requirements.txt
+          venv/bin/pip install pip-tools
+          venv/bin/pip-sync
```

Comment:
we should be using `pip-sync` instead of `pip install`, so that dependencies can be both installed and uninstalled here.